### DISCUSSION TITLE
设置CPU使用率的工具-cpuocup

### DISCUSSION THREAD

**Discussion by giscus** [caibingcheng/comment#314] [Announcements]
# 设置CPU使用率的工具-cpuocup

cpuocup是一款设置CPU使用率的工具, 可以设置若干线程的CPU使用率, 可以将线程绑定到对应的CPU核心, 也可以设置线程的执行优先级(需要sudo权限). 在一些需要低效CPU的测试场合, 该工具可能帮得上忙.

https://imcbc.cn/202305/cpuocup/

**Comment by ryan4yin** [comment]
恭喜找到新工作~

**Reply by caibingcheng** [reply]
谢谢，差不多一个半月了已经

**Comment by penndu** [comment]
安装一个试试！

**Reply by caibingcheng** [reply]
用处还是比较小的~~

**Reply by penndu** [reply]
宝塔内置比较直观。
